src/main/java/seedu/address/model/volunteer/Volunteer.java

```diff
@@ -21,6 +25,7 @@ public class Volunteer {
     private final Date availableDate;
     private final Time startTimeAvailability;
     private final Time endTimeAvailability;
+    private final ObservableList<String> involvedIn;
```

Comment:
I might be wrong on this but I had thought we were storing a List<Event> instead of a List<String>. 
If I'm wrong please ignore this comment.

Reply:
Ah, I saved it as an observable list so that when we assign volunteers and events, it would immediately be reflected on the ui

Reply:
Yoo soz, I don't quite get how the JavaFX works. Why does it have to be an observable list? Is it not enough that events is already in an observable list? Is it because of the cell factory?